api/projects/tags/models.py

```diff
@@ -12,6 +16,22 @@ class Tag(AbstractBaseExportableModel):
     description = models.CharField(max_length=512, blank=True, null=True)
     project = models.ForeignKey(Project, on_delete=models.CASCADE, related_name="tags")
 
+    is_system_tag = models.BooleanField(
+        default=False,
+        help_text="Indicates that a tag was created by the system, not the user.",
+    )
```

Comment:
Are we going to allow the users to mark flags as stale?

Reply:
Yes, that's the plan for sure. Not sure if my code here caters for that though. I will check...

Reply:
Ah, well, yes - it will. That's why we're creating the tag on project create. Users will then be able to use that tag on their features.

Reply:
> Yes, that's the plan for sure.

No further questions in that case.

> Not sure if my code here caters for that 

Having seen the workflows PR I believe it does, but feel free to double-check.